g5k_discovery/vue/HardwareCatalogueFilters.vue

```diff
@@ -136,13 +417,83 @@
 .btn-filter {
   margin-bottom: 20px;
 }
+
+.range-slider {
+  position: relative;
+  width: 100%;
+  height: 2em;
+}
+.range-slider input[type="range"] {
+  position: absolute;
+  left: 0; top: 0;
+  width: 100%;
+  height: 2em;
+  background: transparent;
+  pointer-events: auto;
+  margin: 0;
+  padding: 0;
+}
+.range-slider input[type="range"]:focus {
+  outline: none;
+}
+
+.range-slider input[type="range"]::-webkit-slider-thumb {
+  position: relative;
+  z-index: 2;
+}
+.range-slider input[type="range"].range-min {
+  z-index: 3;
+}
+.range-slider input[type="range"].range-max {
+  z-index: 2;
+}
+
+.simple-filter-row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.simple-filter-row > [class*='col-'] {
+  float: none;
+}
+
+.threads-slider {
+  order: 3;
+}
+
+.simple-filter-row > [class*='col-']:not(.threads-slider):nth-child(n+5) {
+  order: 4;
+}
+
 </style>
 
 <script>
 import JSPath from "jspath";
 import { capitalCase, snakeCase } from "change-case";
 import { advancedCapabilities, simpleCapabilities } from "./capabilities";
 
+const PROCESSOR_DETAIL_FIELDS = [
+  "Cache L1", "Cache L2", "Cache L3", "Clock Speed", "Version",
+  "Other Description", "Cache L1 D", "Cache L1 I"
+];
+
+const PLACEMENT_DETAIL_FIELDS = [
+  "Rack"
+];
+
+const SSD_DETAIL_FIELDS = [
+  "Model",
+  "Vendor",
+  "Serial"
+];
+
+const NETWORK_DEVICES_DETAIL_FIELDS = [
+  "Model",
+  "Vendor",
+  "Name",
+  "# Active Devices"
+];
```

Comment:
Do we want to make these configurable?

Reply:
I like having the "detail fields" grouped up here, but I'm not sure it makes sense to make them configurable until we have refined the data model some more.